### ISSUE TITLE
Build workflow stays pending on docs PRs

### ISSUE THREAD

**Issue by pavithraes** [Point72/csp#190]
**Describe the bug**

The `build.yml` workflow is not run for PRs that update docs, but it's required for merging. This causes the check to remain hanging, blocking PR merges (without overrides), see: https://github.com/Point72/csp/pull/189

Ref, see the first note here: https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

**To Reproduce**

Make a PR that updates only documentation.

**Expected behavior**

The check is either skipped by default, or we add docs for how to skip it.

**Additional context**

I don't know if `[skip ci]` ([ref](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs)) skips all checks or only pending ones, will verify on a new PR.

If this works, we can just add a note in the contributing docs. Else, needs further investigation.

**Comment by timkpaine** [comment]
This is intentional behavior
